src/process.rs

```diff
@@ -15,37 +15,28 @@ pub struct Line {
 pub type Graph = Vec<Line>;
 pub type Graphs = HashMap<String, Graph>;
 
-fn fsts<K, V>(hm: &HashMap<K, V>) -> Result<Vec<f64>>
-where
-    K: NumCast + Ord + Copy + Debug,
-{
-    hm.iter()
-        .sorted_by_key(|x| x.0)
-        .map(|x| <f64 as NumCast>::from(*x.0).ok_or(anyhow!("Failed casting {:?} to f64", x.0)))
-        .collect()
-}
-fn snds<K, V>(hm: &HashMap<K, V>) -> Result<Vec<f64>>
-where
-    K: Ord,
-    V: NumCast + Copy + Debug,
-{
-    hm.iter()
-        .sorted_by_key(|x| x.0)
-        .map(|x| <f64 as NumCast>::from(*x.1).ok_or(anyhow!("Failed casting {:?} to f64", x.1)))
-        .collect()
-}
```

Comment:
Correct me if I'm wrong but both functions do exactly the same no, just on the key or value respectively?